src/02 - Object types.md

```diff
@@ -29,559 +32,560 @@ The following notation is used in the tables below:
+| M107      | date   | arkivperiodeStartDato             |          |             |                |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                                                                |
+| M108      | date   | arkivperiodeSluttDato             |          |             |                |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                                                                |
+| M500      | string | skjerming                         |          |             | skjerming      |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                                                                |
+|           |        |                                   |          |             |                |  |      |      |       |      |      |        |                                                                                                                                                                                |
+|           | ref    | refArkiv                          |    x     |             |                |  |      |  x   |   x   |      |  x   |   x    |                                                                                                                                                                                |
+|           | ref    | refPrimaerKlassifikasjonssystem   |          |             |                |  |      |  x   |   x   |      |  x   |   x    |                                                                                                                                                                                |
+|           | ref    | refSekundaerKlassifikasjonssystem |          |             |                |  |      |      |   x   |      |  x   |   x    |                                                                                                                                                                                |
+|           | ref    | refMappe                          |          |             |                |  |      |      |   x   |      |  x   |   x    | <p>&bull; references **Mappe** or **Saksmappe**</p>                                                                                                                            |
+|           | ref    | refRegistrering                   |          |             |                |  |      |      |   x   |      |  x   |   x    | <p>&bull; references **Basisregistrering** or **Journalpost** </p> <p>&bull; references to **Basisregistrering** and **Journalpost** not currently supported in Documaster</p> |
```

Comment:
Can also reference `Moeteregistrering` although not currently supported.

References to `Basisregistrering` **are supported**.

Reply:
No need to document a reference which is not currently supported.